core/templates/pages/contributor-dashboard-page/modal-templates/question-suggestion-review.component.html

```diff
@@ -70,7 +70,7 @@ <h3 class="oppia-mobile-question-header" title="{{questionHeader}}">
       <button class="btn btn-danger e2e-test-question-suggestion-review-reject-button"
               *ngIf="reviewable"
               [disabled]="reviewMessage.length == 0"
-              (click)="reject(reviewMessage)">Reject
+              (click)="reject()">Reject
```

Comment:
Hey @siddharth-singh-bhadouria , would you please explain this change?

Reply:
Actually by mistake i pushed some changes of another issue in this branch. 

Will it be fine if i revert the unnecessary changes or will i need to create a new PR?

Reply:
@siddharth-singh-bhadouria  you can try reverting the changes in this PR itself. Also if you need a review from a reviewer, you will have to assign them to the PR by adding a comment `PTAL @{reviewer_username}`